src/main/resources/org/aerogear/kryptowire/workflow/KWSubmitStep/config.jelly

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:c="/lib/credentials">
+    <f:entry field="platform" title="Platform">
+        <f:textbox/>
```

Comment:
I think we can use a select field instead of a input one:

```
<select name="platform">
            <option value="android">Android</option>
            <option value="ios">iOS</option>
        </select>
```

Reply:
:+1: